cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/RecoverFromStorageCommand.java

```diff
@@ -526,7 +536,8 @@ private boolean allowSegment(String segmentName) {
         //    of recovery, when container comes up we update the lenghts in the metadata.
         // 3. Same as scaleGroup for COMMIT_STREAM_READERS
         if (segmentName.contains(RG_SCALE_GROUP) || segmentName.contains(EVENT_PROCESSEOR_SEGMENT)
-            || segmentName.contains(COMMIT_STREAM_READERS)) {
+            || segmentName.contains(COMMIT_STREAM_READERS) || segmentName.contains(ABORT_STREAM_READERS)
+            || segmentName.contains(KVT_STREAM_READERS)) {
             return false;
```

Comment:
for the same reason as mentioned in the comment above for RGscale to avoid stale state recovery. In the future versions we would be specifically altering the state that blocks reads and remove these from here.